modules/ztp-migrate-clusterinstance-overview.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-migrate-clusterinstance.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-migrate-clusterinstance-overview_{context}"]
+= Overview of migrating from SiteConfig CRs to ClusterInstance CRs
+
+The `ClusterInstance` CR provides a more unified and generic approach to defining clusters and is the preferred method for managing cluster deployments in the {ztp} workflow.
+The SiteConfig Operator, which manages the `ClusterInstance` custom resource (CR), is a fully developed controller shipped as an add-on within {rh-rhacm-first}.
+
+[IMPORTANT]
+====
+The SiteConfig Operator only reconciles updates for `ClusterInstance` objects. The controller does not monitor or manage deprecated `SiteConfig` objects.
+
+Use either `SiteConfig` or `ClusterInstance` CRs for cluster definitions in the hub cluster to avoid unintended deployments of managed clusters.
+====
+
+The migration from `SiteConfig` CRs to `ClusterInstance` CRs provides several improvements, such as enhanced scalability and a clear separation of cluster parameters from the cluster deployment method. For more information about these improvements, and the SiteConfig Operator, see link:https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.12/html-single/multicluster_engine_operator_with_red_hat_advanced_cluster_management/index#siteconfig-intro[SiteConfig].
+
+The migration process involves the following high-level steps:
+
+. Create a parallel pipeline by preparing a new Git folder in your repository and creating a new Argo CD project and application that monitors this folder.
```

Comment:
I would say, preparing a new Git folder structure. Eventually, you will create a Git structure with all the elements required in your ZTP pipeline. See https://labs.sysdeseng.com/5g-ran-deployments-on-ocp-lab/4.19/deployment-considerations.html#git-repo-structure